src/services/services.ts

```diff
@@ -2101,7 +2101,7 @@ namespace ts {
         };
     }
 
-    export function preProcessFile(sourceText: string, readImportFiles = true): PreProcessedFileInfo {
+    export function preProcessFile(sourceText: string, readImportFiles = true, isJavaScript = false): PreProcessedFileInfo {

```

Comment:
i thought we are going to default this to true, then change it later to false from the VS side..